nifi-nar-bundles/nifi-hive-bundle/nifi-hive3-processors/pom.xml

```diff
@@ -412,6 +412,15 @@
     </dependencies>
     <build>
         <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <environmentVariables>
+                        <JDK_JAVA_OPTIONS>--add-opens java.base/java.net=ALL-UNNAMED</JDK_JAVA_OPTIONS>
```

Comment:
A comment on the reason for this inclusion avoid be helpful, mentioning the offending Hive class.

Reply:
added comment